include/FNA3D.h

```diff
@@ -132,7 +132,10 @@ typedef enum FNA3D_SurfaceFormat
 	FNA3D_SURFACEFORMAT_HALFVECTOR2,
 	FNA3D_SURFACEFORMAT_HALFVECTOR4,
 	FNA3D_SURFACEFORMAT_HDRBLENDABLE,
-	FNA3D_SURFACEFORMAT_COLORBGRA_EXT
+	FNA3D_SURFACEFORMAT_COLORBGRA_EXT,
+	FNA3D_SURFACEFORMAT_RGB8ETC2,
```

Comment:
my preference would be for ETC2 to be at the front of the format name, not the end